packages/neuron-wallet/tests/services/multisig.test.ts

```diff
@@ -77,8 +77,14 @@ describe('multisig service', () => {
 
   describe('save multisig config', () => {
     it('has exist', async () => {
-      await expect(multisigService.saveMultisigConfig(defaultMultisigConfig)).rejects.toThrow()
+      const result = await getConnection()
```

Comment:
I don't get the point of this change, in the left, it expect to throw an exception when calling `saveMultisigConfig`.
But on the right, it does not call any function in multisig service.

Reply:
Same question, duplicated `multisig config` should be rejected

Reply:
Updated.